clusterloader2/pkg/measurement/common/slos/windows_node_resource_usage.go

```diff
@@ -31,13 +31,33 @@ import (
 const (
 	windowsResourceUsagePrometheusMeasurementName = "WindowsResourceUsagePrometheus"
 	// get top 10 non-system processes with highest cpu usage within 1min query window size
-	cpuUsageQueryTop10 = `topk(10, sum by (process) (irate(wmi_process_cpu_time_total{process!~"Idle|Total|System"}[5m]) / on(job) group_left wmi_cs_logical_processors) * 100)`
+	cpuUsageQueryTop10 = `topk(10, sum by (process) (irate(windows_cpu_time_total{process!~"Idle|Total|System"}[5m]) / on(job) group_left windows_cs_logical_processors) * 100)`
```

Comment:
This was 10 ten processes not just cpu, I think knowing top process would be valuable.

Reply:
Thank you for noticing this. Initially when I changed the name of this metric I didn't have the process collector enabled and I could only find windows_cpu_time_total metric
Going to submit a change soon with the correct metric name.